bot.py

```diff
@@ -116,8 +116,10 @@ def sendmail(self):
             msg['Subject'] = self.subject
             msg['From'] = self.fromm
             msg['To'] = self.mail_to
-
-            smtp = smtplib.SMTP('localhost')
```

Comment:
more pythonic to do
smtp_server = os.environ.get('SMTP_SERVER', 'localhost')
 :)